src/uct/ib/dc/dc_mlx5.h

```diff
@@ -50,6 +50,18 @@ struct ibv_ravh {
         _txwq = &(_iface)->tx.dcis[_dci].txwq; \
     }
 
+#define UCT_DC_MLX5_CHECK_FORCE_FULL_HANDSHAKE(_self, _config, _config_name, \
```

Comment:
smth like
/* Set iface flag for enabling full handshake on DCI/DCT according user configuration.
   Fail if forcing full-handlshake on a HW that does not support it. */